leobot_description/package.xml

```diff
@@ -7,4 +7,6 @@
   <license>MIT</license>
 
   <buildtool_depend>catkin</buildtool_depend>
+  <run_depend>sensor_msgs</run_depend>
+  <build_depend>urg_node</build_depend>
```

Comment:
This node is not needed for simulation. Please remove it.

Reply:
@AndriyPt sensor_msg is removed

Reply:
urg_node as well :)